client/shipping-label/views/purchase/steps/packages/add-item.js

```diff
@@ -72,7 +72,7 @@ const AddItemDialog = ( {
 				<h1 className="form-section-heading">{ __( 'Add item' ) }</h1>
 				<div className="wcc-label-packages-dialog__body">
 					<p>
-						{ __( 'Which item would you like to add to {{pckg/}}?', {
+						{ __( 'Which items would you like to add to {{pckg/}}?', {
```

Comment:
I'm not sure but I feel like if we're explicitly acknowledging that we know there's only one item, it doesn't make sense to proceed to ask which one, and in that case we should go further — like reformulating the prompt as a confirmation?